backend/src/db/crud/survey.py

```diff
@@ -5,17 +5,41 @@
 
 
 def get_survey_by_code(survey_code: str, session: Session) -> Survey:
-    statement = select(Survey).where(Survey.survey_code == survey_code)
+    statement = select(Survey).where(
+        (Survey.survey_code == survey_code)
+        & (Survey.is_deleted == False)  # noqa: E712
```

Comment:
(Survey.is_deleted is False) should work (without noqa)

Reply:
Doesn't work, query returns no surveys, tried that before.

Reply:
ok, thats fine then